tests/post/steps/test_volume.py

```diff
@@ -255,6 +285,24 @@ def _check_status():
         name='checking error for Volume {}'.format(name)
     )
 
+
+@then(parsers.parse("the Volume '{name}' is marked for deletion"))
+def check_volume_deletion_timestamp(name, k8s_custom_client):
+    def _check_deletion_timestamp():
+        volume = _get_volume(k8s_custom_client, name)
+        if volume is None:
+            assert False, 'Volume {} not found'.format(name)
+
+        assert volume['metadata']['deletionTimestamp'] is not None,\
+            'Unexpected deletion timestamp got {}'.format(
+                volume['metadata']['deletionTimestamp']
+            )
+
+    utils.retry(
+        _check_deletion_timestamp, times=30, wait=2,
```

Comment:
This should appear quite fast (we don't need to call Salt or whatever) so we can reduce the settings here: `times=10, wait=1` or `times=10, wait=2`